src/main/java/ro/itschool/ema/models/dtos/OrganizerDTO.java

```diff
@@ -0,0 +1,33 @@
+package ro.itschool.ema.models.dtos;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Past;
+import lombok.Data;
+import org.springframework.validation.annotation.Validated;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Data
+@Validated
+public class OrganizerDTO implements Serializable {
```

Comment:
I thought that the organizer means the organization, not a being having a date of birth. It is like a company organizing events. Let s have a chat about it